src/components/frame/Extensions/pages/LazyConsensusPage/LazyConsensusPage.tsx

```diff
@@ -45,67 +45,73 @@ const LazyConsensusPage: FC = () => {
     extensionData?.extensionId === Extension.VotingReputation;
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <Spinner loadingText={{ id: 'loading.extensionsPage' }}>
-        <ThreeColumns
-          leftAside={<Navigation />}
-          topRow={
-            <div className="flex justify-between flex-col flex-wrap sm:items-center sm:flex-row sm:gap-6">
-              <div className="flex flex-wrap gap-4 flex-col w-full sm:flex-row sm:items-center md:gap-8">
-                <ActionButtons
-                  extensionData={extensionData}
-                  extensionStatusMode="governance"
-                  extensionStatusText={formatMessage({
-                    id: 'status.governance',
-                  })}
-                />
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Spinner loadingText={{ id: 'loading.extensionsPage' }}>
+          <ThreeColumns
+            leftAside={<Navigation />}
+            topRow={
+              <div className="flex justify-between flex-col flex-wrap sm:items-center sm:flex-row sm:gap-6">
+                <div className="flex flex-wrap gap-4 flex-col w-full sm:flex-row sm:items-center md:gap-8">
+                  <ActionButtons
+                    extensionData={extensionData}
+                    extensionStatusMode="governance"
+                    extensionStatusText={formatMessage({
+                      id: 'status.governance',
+                    })}
+                  />
 
-                <div className="flex gap-6 items-center justify-end min-h-[2rem]">
-                  {isEnableButtonVisible && (
-                    <Button
-                      mode="primarySolid"
-                      type="submit"
-                      isFullSize={isMobile}
-                    >
-                      {formatMessage({ id: 'button.enable' })}
-                    </Button>
-                  )}
+                  <div className="flex gap-6 items-center justify-end min-h-[2rem]">
+                    {isEnableButtonVisible && (
+                      <Button
+                        mode="primarySolid"
```

Comment:
You don't have to pass `primarySolid` to the mode prop because it's setting by default in the Button component.